h/api/models/document.py

```diff
@@ -37,7 +37,7 @@ def title(self):
         titles = [m.value for m in self.meta if m.type == 'title']
 
         if titles:
-            return titles[0]
+            return titles[0][0]
```

Comment:
I'm not sure what the model should be here:

1. _Should_ the DocumentMeta model have an array for the value column as it does? Does it have to? If not we can change the model to just have a string

2. If it does have to be an array, is there always at least one item in the array? Or we have to watch our for empty lists? Does it ever make sense  to have a DocumentMeta like this:

   ```
   type: "title",
   value: [],
   ...
   ```

We should probably add a `title` or maybe `first_title` property to the `DocumentMeta` class to return either the first title or an empty string